warehouse/models/payments_views/payments_rides.sql

```diff
@@ -294,11 +294,7 @@ join_table AS (
         t2.latitude AS off_latitude,
         t2.longitude AS off_longitude,
         t2.geography AS off_geography,
-        CASE
-            WHEN
-                t1.route_id != 'Route Z' THEN t1.route_id
-            ELSE COALESCE(t2.route_id, 'Route Z')
-        END AS route_id
+        COALESCE(t1.route_id, t2.route_id, Null) AS route_id
```

Comment:
nit: you don't need a final `null`, that's already what coalesce will end up with if all inputs are null